_sass/elements/_tables.scss

```diff
@@ -54,6 +54,7 @@ td {
 
   thead {
     background-color: transparent;
+    line-height: 1.2;
```

Comment:
This could be `line-height: $light-height-caps;`

It will compute to `line-height: 1.2rem;`